<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskHost</class>
 <widget class="QDialog" name="AskHost">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>195</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter Ruby app's address</string>
  </property>
  <widget class="QLineEdit" name="commandsPort">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>111</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5556</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eventsPort">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>111</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5555</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
    </font>
   </property>
   <property name="text">
    <string>Command Port</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>38</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="serverHost">
      <property name="font">
       <font>
        <family>Lucida Grande</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>tcp://127.0.0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="serverIpAddressLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>14</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
    </font>
   </property>
   <property name="text">
    <string>Enter the IP address of the server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
    </font>
   </property>
   <property name="text">
    <string>Data Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hostResetButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Reset to Defaults</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hostSelectedButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>94</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
    </font>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>hostSelected()</slot>
  <slot>hostReset()</slot>
 </slots>
</ui>
